.github/workflows/deploy-docs.yaml

```
@@ -4,12 +4,15 @@ on:
   push:
     branches:
       - main
+      - rc/*
```

Did you mean `beta`?

```suggestion
      - beta/*
```

---

Fixed in https://github.com/autowarefoundation/autoware-documentation/pull/244/commits/a03a6ffe963743252d7850e5153a619c17988ab1